leaderboard-page/client/src/App.vue

```diff
@@ -56,19 +59,28 @@ export default {
       headers: [
         {
           text: "",
-          value: ""
+          value: "",
+          align: "center",
+        },
+        {
+          text: "",
+          value: "",
+          sortable: false
         },
         {
           text: "Login",
-          value: "UserLogin"
+          value: "UserLogin",
+          align: "ceenter"
```

Comment:
`ceenter`

Reply:
oh that's embarrassing! I'll get that fixed up right away!